.gitattributes

```diff
@@ -0,0 +1,4 @@
+*        text=auto
+*.go     text      eol=lf
+*.mod    text      eol=lf
+*.sum    text      eol=lf linguist-generated=true
```

Comment:
This tells git how to handle golang files in windows and which files it should hide in the github diff